process/sync/baseSync.go

```diff
@@ -27,11 +27,14 @@ const sleepTime = 5 * time.Millisecond
 
 // maxRoundsToWait defines the maximum rounds to wait, when bootstrapping, after which the node will add an empty
 // block through recovery mechanism, if its block request is not resolved and no new block header is received meantime
-const maxRoundsToWait = 5
+const maxRoundsToWait = 3
 
 // maxHeadersToRequestInAdvance defines the maximum number of headers which will be requested in advance if they are missing
 const maxHeadersToRequestInAdvance = 10
 
+// roundsMultipleStep defines the multiple step of rounds on which nodes could take some decisions
```

Comment:
some better explenation, and rename variable

Reply:
Done